docs/index.html

```diff
@@ -159,7 +159,7 @@
 
     <!--Register Service Worker-->
     <script>
-        if ('serviceWorker' in navigator) {
+        if ('serviceWorker' in navigator && window.location.hash !== "#dev") {
```

Comment:
```suggestion
        // We disable caching during development so that we always view the latest version.
        if ('serviceWorker' in navigator && window.location.hash !== "#dev") {
```

Reply:
I've added it.